python/dgl/data/utils.py

```diff
@@ -237,7 +237,7 @@ def check_sha1(filename, sha1_hash):
     return sha1.hexdigest() == sha1_hash
 
 
-def extract_archive(file, target_dir, overwrite=False):
```

Comment:
Why changes the default value? Will this break existing usage?

Reply:
The extract_archive is used after the download function like in 
https://github.com/dmlc/dgl/blob/8275bc29436fa0d2967d6505c0fc1e4383cf1137/python/dgl/data/movielens.py#L255
I think because the download function downloads the new archive file, we should extract the file and overwrite the old one by default. Otherwise, we only download the new archive file but don't update it if the target_dir exists.
`if os.path.exists(target_dir) and not overwrite:`

I think it won't break existing usage, and the comment of extract_archive function also say it is true by default.
https://github.com/dmlc/dgl/blob/8275bc29436fa0d2967d6505c0fc1e4383cf1137/python/dgl/data/utils.py#L249